### DISCUSSION TITLE
script mode: theme option

### DISCUSSION THREAD

**Discussion by numberokta** [davatorium/rofi#1881] [Q&A] [answered]
hi, in **script mode**, **theme option** just works in **initial state** when **ROFI_RETV** is 0.
i dont know its bug or just work this way ?

**Comment by DaveDavenport** [comment, marked as answer]
? no idea what you are doing, or how you are doing it.  You are not providing much info.

It works fine for me: https://github.com/davatorium/rofi/blob/next/Examples/test_script_mode_color.sh

The things you can set using this are very limited, rofi does not completely rebuild the interface, so you can only change things that are redrawn/reloaded.

**Reply by numberokta** [reply]
I want to change enabled or text of element.

**Reply by numberokta** [reply]
It works fine for changing color. As you said, it's very limited.

**Reply by DaveDavenport** [reply]
Yeah, its a bit of a tradeoff in flexibility and speed/redrawing artifacts.
